### PR TITLE
Some clarifications

### PR DESCRIPTION
**Pull request by mauroservienti**
@SeanFeldman @MarcinHoppe please review

### CODE REVIEW

**Comment by MarcinHoppe** [comment]
@mauroservienti @SeanFeldman Good additions. I somewhat improved wording and rearranged the things s little bit. Please let me know if looks good and I will merge.

File: README.md [hunk 1]
```diff
@@ -16,8 +16,14 @@ Please refer to documentation of each individual component for download and setu
 
 ### RabbitMQ connection string
 
-RabbitMQ connection string can be set using `Backend.RabbitMqConnectionString.Value` property in the `Shared` project.
+RabbitMQ connection string can be configured by setting the `Value` property of the `Backend.RabbitMqConnectionString` class in the `Shared` project.
```
Comment (SeanFeldman) [inline README.md:19]: > `Backend.RabbitMqConnectionString`  class in the `Shared` project.

@mauroservienti this is my re-factoring mess :disappointed: Should we move that to `Shared` namespace?

Comment (MarcinHoppe) [inline README.md:19]: @SeanFeldman I fixed it on all branches. Can you check it compiles and still works?

**Comment by mauroservienti** [comment]
OK, I like it

**Pull request merged**